### PR TITLE
Test/hydration test

### PR DESCRIPTION
**Pull request by Jedeo**
What’s this PR do?
- added tests for methods in User class
-  it( 'should return the average fluid ounces consumed per day for all time', () => {} )
-  it('should return fluid ounces they consumed for a specific day (identified by a date)' () => {} )
-  it(should return fluid ounces of water consumed each day over the course of a week (7 days)', () => {} )

Where should the reviewer start?
- the hydration test file

close()

### CODE REVIEW

File: test/Hydration-test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { expect } from "chai";
+///import Hydration from "../src/Hydration";
```
Comment (forsethnico) [inline test/Hydration-test.js:2]: Remove this commented out code eventually

File: test/Hydration-test.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        date: "2019/06/16",
+        numOunces: 69,
+      },
+    ];
+    data = userTestData;
+    user = new User(data[0]);
+  });
+
+  it("should return the average fluid ounces consumed per day for all time", () => {
+    expect(user.calAverageFluid(hydrationData, 1)).to.equal(204);
```
Comment (piperjarrett) [inline test/Hydration-test.js:48]: needs to be  equal to the average

Comment (forsethnico) [inline test/Hydration-test.js:48]: Update to use the averaged value as comparison

File: test/Hydration-test.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  });
+
+  it("should return fluid ounces of water consumed each day over the course of a week (7 days)", () => {
+    console.log(user.getWeeklyFluids(hydrationData));
+    expect(user.getWeeklyFluids(hydrationData)).to.deep.equal([
+      { date: "2019/06/15", numOunces: 37 },
+      { date: "2019/06/15", numOunces: 37 },
+      { date: "2019/06/18", numOunces: 61 },
+      { date: "2019/06/16", numOunces: 69 },
+    ]);
```
Comment (piperjarrett) [inline test/Hydration-test.js:62]: needs to go for seven days exactly

File: test/Hydration-test.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  it("should return fluid ounces they consumed for a specific day (identified by a date)", () => {
+    expect(user.getDayFluid(hydrationData, "2019/06/15")).to.equal(74);
+  });
+
+  it("should return fluid ounces of water consumed each day over the course of a week (7 days)", () => {
+    console.log(user.getWeeklyFluids(hydrationData));
+    expect(user.getWeeklyFluids(hydrationData)).to.deep.equal([
+      { date: "2019/06/15", numOunces: 37 },
+      { date: "2019/06/15", numOunces: 37 },
+      { date: "2019/06/18", numOunces: 61 },
```
Comment (forsethnico) [inline test/Hydration-test.js:60]: Create data set that has 7 consecutive data points of water drinking

**Review by forsethnico — APPROVED** [review]
A few of the testing comparisons need to be updated for averaging and 7 days timeframe

**Review by piperjarrett — COMMENTED** [review]
Looks good! just add the testing for the changes you need to make in in the User class.

**Pull request merged**